### ISSUE TITLE
How to understand param --padding_constant && --segm_downsampling_rate

### ISSUE THREAD

**Issue by JonyFaker** [CSAILVision/semantic-segmentation-pytorch#82]
How to understand param **--padding_constant && --segm_downsampling_rate？**
Besides, what if **i want to replace encoder network you provided**？I try to use mobilenet as an encoder，but，when training，size of pred and target are not match。 what should i do to solve the problem and adapt my own encoder network？

hope your response，Thanks！

@metalbubble @quantombone

**Comment by Tete-Xiao** [comment]
Padding constant is the maximum down-sampling rate of the neural networks. For example, a ResNet w/o dilated conv has padding constant 32.

Segmentation downsampling rate is the down-sampling rate of ground-truth segmentation map when calculating loss. For example, UPerNet calculates such loss on a map with down-sampling rate 4.

**Comment by Jason-xin** [comment]
@Tete-Xiao The two params must be set in testing?
